idseq/uploader.py

```diff
@@ -153,12 +153,7 @@ def upload(sample_name, project_id, headers, url, r1, r2, host_genome_name, chun
     # Get version of CLI from setuptools
     version = pkg_resources.require("idseq")[0].version
 
-    # Send metadata as sample_name => {metadata_key: value}
-    csv_data = {}
-    with open(metadata_file) as file_data:
-        for row in list(csv.DictReader(file_data)):
-            name = row.pop("sample_name")
-            csv_data[name] = row
+    host_genome_name = get_match_in_dict(["host_genome", "Host genome", "Host Genome", "host genome"], csv_metadata)
```

Comment:
Right now, the metadata endpoints just check for "host_genome" and "Host Genome", not "host genome" or "Host genome". We should make the same everywhere for consistency.